tests/Doctrine/Tests/Common/Cache/MemcachedCacheTest.php

```diff
@@ -51,9 +51,8 @@ public function testGetMemcachedReturnsInstanceOfMemcached()
 
     public function testContainsWithKeyWithFalseAsValue()
     {
-        $testKey = __CLASS__.'#'.__METHOD__;
+        $testKey = __METHOD__;
         $driver = $this->_getCacheDriver();
-
         $this->memcached->set($testKey, false);
```

Comment:
I prop. need to fix the NS stuff now I no longer directly test the cache implementation but the public API layer (as requested), thanks for looking!